homeassistant/components/epson/media_player.py

```diff
@@ -123,7 +123,11 @@ async def set_unique_id(self) -> bool:
 
     async def async_update(self) -> None:
         """Update state of device."""
-        power_state = await self._projector.get_power()
+        try:
+            power_state = await self._projector.get_power()
+        except ProjectorUnavailableError as ex:
```

Comment:
You should either log `ex` or don't use `as ex` at all. I prefer logging

Reply:
I think we can log this at the debug level as the following. There is no point in logging it at any higher level, since then we will again spam all the HA logs.

```suggestion
        except ProjectorUnavailableError as ex:
            _LOGGER.debug("Projector is unavailable: %s", ex)
```